deployment/build-output.txt

```diff
@@ -0,0 +1,25 @@
+template_dir: /Users/ianwow/development/aws-media-insights-engine/deployment
```

Comment:
Did you intend to include the build output in this pull request?

Reply:
oh, no. good catch. thanks.